pkg/controller/installation/installation_controller.go

```diff
@@ -644,7 +647,7 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 		installation.Status.PreflightStatus = integreatlyv1alpha1.PreflightFail
 		installation.Status.PreflightMessage = "Spec.useClusterStorage must be set to either 'true' or 'false' to continue"
 		_ = r.client.Status().Update(context.TODO(), installation)
-		logrus.Infof("preflight checks failed on useClusterStorage value")
+		log.Info("preflight checks failed on useClusterStorage value")
```

Comment:
Wonder if this could be marked as an error as the installation won't continue without it?

Reply:
ah, nevermind, I see there's no error available.

Reply:
hmm, maybe use of r.Log.Warning would be better?

Reply:
Changed to warning